geopandas/_vectorized.py

```diff
@@ -747,6 +747,18 @@ def minimum_bounding_radius(data):
         )
 
 
+def segmentize(data, max_segment_length):
+    if compat.USE_SHAPELY_20:
+        return shapely.segmentize(data, max_segment_length)
+    elif compat.USE_PYGEOS:
+        return pygeos.segmentize(data, max_segment_length)
+    else:
+        raise NotImplementedError(
+            f"shapely >= 2.0 or PyGEOS is required, "
```

Comment:
```suggestion
            "shapely >= 2.0 or PyGEOS is required, "
```

Reply:
We should merge #2921, it would catch (and fix) this kind suggestions.